bin/bulk_load

```diff
@@ -7,15 +7,12 @@ LIBRARY_PATH = PROJECT_ROOT + "lib/"
 
 $LOAD_PATH << LIBRARY_PATH unless $LOAD_PATH.include?(LIBRARY_PATH)
 
-INDICES = %w{mainstream detailed government}.freeze
-
 require "rummager"
 
 Slop.parse(help: true) do
-  search_config = SearchConfig.new
-  banner %{Usage: #{File.basename(__FILE__)} {#{search_config.index_names.join("|")}}
+  banner %{Usage: #{File.basename(__FILE__)}
 
-Bulk loads data from stdin to the specified index. The data should be in the
+Bulk loads data from stdin to the page traffic index. The data should be in the
```

Comment:
I think it'd be worth renaming this script now that it just handles page traffic data. "bulk_load" makes it sound more generic than it is.

Reply:
Done, I have also added a sym link for the old file name until we update the script in puppet